New Bank/NewBank/newbank/server/NewBank.java

```diff
@@ -53,9 +62,25 @@ public synchronized String processRequest(CustomerID customer, String request) {
 				String to = input[3];
 				return moveFunds(customer,amount,from,to);
 			}
+			if(input[0].equals("PAY")) {
+				if(input.length < 4) { // return fail if not enough information is provided
```

Comment:
This length check of 4 is shorter than the expected length of the user input for PAY. So it would pass the check if the length was 4 but then it will have nothing to assign from input[4] onwards.

Reply:
done